content/src/Blip.Api.Template/appsettings.json

```diff
@@ -1,22 +1,22 @@
 ﻿{
-  "Logging": {
-    "IncludeScopes": false,
-    "Debug": {
-      "LogLevel": {
-        "Default": "Warning"
-      }
-    },
-    "Console": {
-      "LogLevel": {
-        "Default": "Warning"
-      }
-    }
-  },
   "Settings": {
     "BlipBotSettings": {
-      "Identifier": "botBlipIdentifier",
-      "AccessKey": "botBlipAccessKey",
-      "Authorization" : "botAuthorizationKey" 
+      "Identifier": "",
+      "AccessKey": ""
+    },
+    "ApplicationName": "",
```

Comment:
Set it to "Blip.Api.Template", that way it will be replaced upon `dotnet new blip-api`